roxie/ccd/ccdlistener.cpp

```diff
@@ -920,6 +920,9 @@ extern void updateAffinity(unsigned __int64 affinity)
                 CPU_SET(core, &cpus);
             affinity >>= 1;
         }
+
+        //MORE: I think this only sets the affinity of the process, not of the threads.
+        //It would require code to iterate through /proc/<pid>/task/*
```

Comment:
That might not be what we want either, since we have support for setting per-query affinity too. Not sure how control:setAffinity interacts/should interact with that.

Reply:
I agree a bit tricky to have different overall and per query affinities concurrently.